_episodes/11-act-actel.md

```diff
@@ -259,10 +289,297 @@ actel_explore_output <- explore(datapack=actel_project,
 Review the file that Actel pops up in our browser. It presumed our Arrays were arranged linearly and alphabetically, which is of course not correct! 
 
 
+### Custom spatial.txt files for Actel
+
+We'll have to tell `Actel` how our arrays are inter-connected. To do this, we'll need to design a spatial.txt file for our detection data.
+
+To help with this, we can go back and visualize our study area interactively, and start to see how the Arrays are connected.
+
+~~~
+# Designing a spatial.txt file -----
+
+library(mapview)
+library(spdplyr)
+library(leaflet)
+library(leafpop)
+
+
+## Exploration - Let's use mapview, since we're going to want to move around, 
+#   drill in and look at our stations
+
+
+# Get a list of spatial objects to plot from actel_spatial_sum:
+our_receivers <- as.data.frame(actel_spatial_sum) %>%    
+  dplyr::filter(Array %in% (actel_spatial_sum %>%   # only look at the arrays already in our spatial file
+                              distinct(Array))$Array)
+
+rcvr_spatial <- our_receivers %>%    
+  dplyr::select(Longitude, Latitude) %>%           # and get a SpatialPoints object to pass to mapview
+  sp::SpatialPoints(CRS('+proj=longlat'))
+# and plot it using mapview. The popupTable() function lets us customize our tooltip
+mapview(rcvr_spatial, popup = popupTable(our_receivers, 
+                                         zcol = c("Array",
+                                                  "Station.name")))  # and make a tooltip we can explore
+
+~~~
+{: .language-r}
+
+Can we design a graph and write it into spatial.txt that fits all these Arrays together? The station value we put in Array for our PROJ61 and PROJ60 projects looks to be a bit too granular for our purposes. Maybe we can combine many arrays that are co-located in open water into a singular 'zone', preserving the complexity of the river systems but creating a large basin to which we can connect the furthest downstream of those river arrays.
+
+To do this, we only need to update the arrays in our spatial.csv file or actel_spatial dataframe. We don't need to edit our source metadata! We will have to define a spatial.txt file and how these newly defined Arrays interconnect. While there won't be time to do that for this example dataset and its large and very complicated region, this approach is definitely suitable for small river systems and even perhaps for multiple river systems feeding a bay and onward to the open water. If you'd like to apply Actel to your data and want to define a custom spatial.txt file, there are some code examples included in code/day2/5_actel_custom_spatial.R that might be helpful to get you started.
+
+## MIGRAMAR Node
```

Comment:
this gets cut?

Reply:
or not til after the fork

Reply:
unclear, depends on testing of the 5_actel_custom_spatial.R file i think?